app/models/ability.rb

```diff
@@ -94,9 +94,11 @@ def initialize(member)
       cannot :destroy, OrderItem, :order => { :member_id => member.id, :completed_at => nil }
 
       # following/unfollowing permissions
-      can :create, Follow do |f|
-        !member.already_following?(f.followed) && f.followed_id != member.id
-      end
+      can :create, Follow
+      cannot :create, Follow, :followed_id => member.id # can't follow yourself
```

Comment:
What will now happen if you type in the address directly and try to follow someone you're already following?